apps/services/personal-representative/src/app/modules/rightTypes/rightTypes.controller.ts

```diff
@@ -103,7 +103,7 @@ export class RightTypesController {
   @ApiOkResponse()
   async removeAsync(
     @Param('code') code: string,
-    @CurrentUser() user: User,
+    @CurrentActor() user: User,
```

Comment:
Why this change? This should only be used in very special circumstances (APIs returning data for the real user even during delegation).

Reply:
Was copying the auth-public-api but will change back